dogen/cli.py

```diff
@@ -69,6 +70,19 @@ def run(self):
         parser.epilog = epilog
         args = parser.parse_args()
 
+        params = {}
+
+        if args.params:
+            for param in args.params:
+                if '=' not in param:
+                    self.log.error("The --param argument with value '%s' could not be parsed. Please make sure you use the 'KEY=value' format" % param)
+                    sys.exit(1)
+
+                k, v = param.split("=", 1)
+                params[k] = v
+
+        args.params = params
```

Comment:
this is very hacky - you should not mess with args object and store your values here.